packages/datagrid/src/datagrid.ts

```diff
@@ -728,6 +728,41 @@ class DataGrid extends Widget {
       this.dataModel instanceof MutableDataModel;
   }
 
+  /**
+   * The renduring context for painting the data grid.
+   */
+  get canvasGC(): CanvasRenderingContext2D {
```

Comment:
Do these need to be `public`?  Would `protected` getters work for your use case?

Reply:
Yes, `protected` is fine.